src/cli/analyze/analyze-cli-command.ts

```diff
@@ -37,6 +37,33 @@ Please follow and contribute to the discussion at:
 		);
 	}
 
+	if (config.format === "webtypes") {
+		// Allow object being passed as JSON from command line
+		if (typeof config.webtypesConfig === "string") {
+			config.webtypesConfig = JSON.parse(config.webtypesConfig);
+		}
+
+		if (!config?.webtypesConfig) throw makeCliError("Missing webtypes-config configuration");
```

Comment:
when is `config` nullish? (the `config?.`)

Reply:
Fixed